docs/mocksupport.rst

```diff
@@ -911,6 +923,29 @@ DMTF CIM schema download support
    .. rubric:: Details
 
 
+
+.. _`config`:
+
+config
+------
+
+.. autoclass:: pywbem_mock.config
+   :members:
+
+   .. rubric:: Methods
+
+   .. autoautosummary:: pywbem_mock.config
+      :methods:
+      :nosignatures:
+
+   .. rubric:: Attributes
+
+   .. autoautosummary:: pywbem_mock.config
+      :attributes:
+
+   .. rubric:: Details
+
```

Comment:
It is a module, not a class. Change the entire `autoclass` statement with all of its sub-statements to this:

```
.. automodule:: pywbem_mock.config
   :members:
```

Reply:
DONE